GeoHealthCheck/init.py

```diff
@@ -68,6 +68,7 @@ def init():
         # Read and override configs
         app.config.from_pyfile('config_main.py')
         app.config.from_pyfile('../instance/config_site.py')
+        app.config.from_envvar('GHC_SETTINGS')
```

Comment:
Think the build failure comes from this line. According to Flask API, IMO should be:
`app.config.from_envvar('GHC_SETTINGS', silent=True)`

https://flask.palletsprojects.com/en/1.1.x/api/#flask.Flask.config:
`silent – set to True if you want silent failure for missing files.`

Reply:
Sorry, I missed that. We should be good to go now.